miso-web/src/main/webapp/scripts/hot_sample.js

```diff
@@ -440,6 +440,7 @@ HotTarget.sample = (function() {
             },
             pack : function(sam, flat, errorHandler) {
               if (!Utils.validation.isEmpty(flat.stainName)) {
+                sam.stain = {};
```

Comment:
Hmm I'd forgotten that they can switch from stained to unstained. This will become more than one line.